docs/guides/automating-a-process-using-bpmn.md

```diff
@@ -0,0 +1,102 @@
+---
+id: automating-a-process-using-bpmn
+title: Automating a process using BPMN
+description: A quickstart to how BPMN can automate your most complex business processes using an easy-to-adopt visual modeling language.
+keywords: [workflow, modeling]
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">Time estimate: 20 minutes</span>
+
+## What is BPMN?
+
+Business Process Model and Notation (BPMN) is the global standard for process modeling. With BPMN, automate your most complex business processes using an easy-to-adopt visual modeling language.
+
+Processes are the algorithms that determine how an organization runs based on independent tasks. Successful businesses grow from proven, effective processes. Therefore, Camunda’s workflow engine executes processes defined in BPMN.
+
+Take the following example where we've outlined a diagram to send an email. Don't worry too much about the symbols as we'll get to that shortly. For now, recognize the start and end of the process, comprised of entering a message, and sending the email.
+
+![sending email bmmn diagram](./img/simple-bpmn-process.png)
+
+Implement control and visibility over your critical business processes, as the workflow engine orchestrates processes that span APIs, microservices, business decisions and rules, human work, IoT devices, RPA bots, and more.
+
+## Set up
+
+At Camunda, we utilize [Modeler](./components/modeler/overview.md) to build out BPMN diagrams.
+
+You can begin building out your processes within Camunda Cloud using Cloud Modeler. Therefore, ensure you’ve [created a Camunda Cloud account](getting-started/create-camunda-cloud-account.md) to get started.
+
+## Getting started with BPMN
+
+Once logged in to your Camunda Cloud account, take the following steps:
+
+1. Click the **Diagrams** tab in the top navigation bar.
+2. Click **Create New Diagram**.
+3. Select the three vertical dots next to **New Diagram** on the top left corner of the page to rename your diagram. In this case, we'll name ours "Bake a Cake."
+
+### BPMN elements
+
+Before we build out the diagram to bake our cake, we need to understand what the components on the left side of the screen signify.
+
+We can build out our process using several elements, including the following:
+
+- Events: The things that happen. For example, start and end events which begin and terminate our process.
+- Tasks: For example, user tasks for a particular user to complete, or service tasks to invoke various webservices.
```

Comment:
We do use `elements` throughout our documentation [here](https://docs.camunda.org/manual/7.16/reference/bpmn20/), and I'd like to avoid `components` as it may cause confusion among our current documentation framework. Thoughts on preference here?